src/server/utils/cassandra.js

```diff
@@ -1,10 +1,11 @@
 import Promise from 'bluebird';
 import config from 'config';
-import { Client, types as CassandraTypes } from 'cassandra-driver';
+import { Client, types as CassandraTypes } from 'dse-driver';
 import { lookupServiceAsync } from './lookup-service';
 
 // Client promises by keyspace
 const clientPromises = new Map();
+const dse = require('dse-driver');
```

Comment:
You shouldn't need to require dse-driver again here (it's already being imported above). Looks like you need the `auth` object imported also so the import statement should be:

```js
import { Client, auth, types as CassandraTypes } from 'dse-driver';
```